sdk/packages/core/src/api/validateOrder.ts

```diff
@@ -7,20 +8,37 @@ import { type Order, isContractCall } from '../types/order.js'
 import { getApiUrl } from '../utils/getApiUrl.js'
 import { toJSON } from '../utils/toJSON.js'
 
+const acceptedResponseSchema = z.object({
+  accepted: z.literal(true),
+  rejectCode: z.literal(0).optional(),
+  rejected: z.literal(false).optional(),
+  rejectReason: z.literal("").optional(),
+  rejectDescription: z.literal("").optional(),
+})
+
+const rejectedResponseSchema = z.object({
+  accepted: z.literal(false).optional(),
+  rejectCode: z.number().optional(),
+  rejected: z.literal(true),
+  rejectReason: z.string(),
+  rejectDescription: z.string(),
+}) 
+
+const errorResponseSchema = z.object({
+  error: z.object({
+    code: z.number(),
+    message: z.string(),
+  }),
+})
+
```

Comment:
- I found `rejectCode`in api response, so I added it
- I cannot find the example of error response, so `errorResponseSchema` could have more schema